### PR TITLE
Fix chronos client initialization

### PR DESCRIPTION
**Pull request by theosotr**
Provide the missing keyword arguments during chronos client
initialization based on the runner parameters.

### CODE REVIEW

**Comment by jmchilton** [comment]
@galaxybot test this

File: lib/galaxy/jobs/runners/chronos.py [hunk 1]
```diff
@@ -115,8 +115,13 @@ def __init__(self, app, nworkers, **kwargs):
             kwargs[self.RUNNER_PARAM_SPEC_KEY] = {}
         kwargs[self.RUNNER_PARAM_SPEC_KEY].update(self.RUNNER_PARAM_SPEC)
         super(ChronosJobRunner, self).__init__(app, nworkers, **kwargs)
+        protocol = 'http' if self.runner_params.get('insecure', True)\
```
Comment (bgruening) [inline lib/galaxy/jobs/runners/chronos.py]: Can you put this into one line? We don't follow the 80vchars pep8 rule :)

File: lib/galaxy/jobs/runners/chronos.py [hunk 2]
```diff
@@ -115,8 +115,13 @@ def __init__(self, app, nworkers, **kwargs):
...
+            else 'https'
         self._chronos_client = chronos.connect(
-            self.runner_params['chronos'])
+            self.runner_params['chronos'],
+            username=self.runner_params.get('username'),
```
Comment (bgruening) [inline lib/galaxy/jobs/runners/chronos.py]: What happens if the username and password is not set? Should we default here so something? `.get('username', None)`

File: lib/galaxy/jobs/runners/chronos.py (hunk 1 above)
Comment (theosotr) [reply lib/galaxy/jobs/runners/chronos.py]: OK

File: lib/galaxy/jobs/runners/chronos.py (hunk 2 above)
Comment (theosotr) [reply lib/galaxy/jobs/runners/chronos.py]: `get(key)` returns `None` by default, if the key is not found, so there is no problem. :)

**Comment by dannon** [comment]
@theosotr Can you add the 'insecure' option to job_conf.xml.sample_advanced?  Then I think this is good to go.

**Pull request merged**